docs/ja/docs/tutorial/testing.md

```diff
@@ -0,0 +1,144 @@
+# テスト
+
+<a href="https://www.starlette.io/testclient/" class="external-link" target="_blank">Starlette</a> のおかげで、**FastAPI** アプリケーションのテストは簡単で楽しいものになっています。
+
+<a href="http://docs.python-requests.org" class="external-link" target="_blank">Requests</a> がベースなので、非常に使いやすく直感的です。
+
+これを使用すると、**FastAPI** と共に <a href="https://docs.pytest.org/" class="external-link" target="_blank">pytest</a> を直接利用できます。
+
+## `TestClient` を使用
+
+`TestClient` をインポートします。
+
+`TestClient` を作成し、**FastAPI** に渡します。
+
+`test_` から始まる名前の関数を作成します (これは `pytest` の標準的なコンベンションです)。
+
+`requests` と同じ様に `TestClient` オブジェクトを使用します。
+
+チェックしたい Python の標準的な式と共に、シンプルに `assert` 文を記述します。
+
+```Python hl_lines="2  12  15-18"
+{!../../../docs_src/app_testing/tutorial001.py!}
+```
+
+!!! tip "豆知識"
+    テスト関数は `async def` ではなく、通常の `def` であることに注意してください。
+
+    また、クライアントへの呼び出しも通常の呼び出しであり、`await` を使用しません。
+
+    これにより、煩雑にならずに、`pytest` を直接使用できます。
+
+!!! note "技術詳細"
+    `from starlette.testclient import TestClient` も使用できます。
+
+    **FastAPI** は開発者の利便性のために `fastapi.testclient` という `starlette.testclient` と同様なものを提供しています。ただし、Startletteから直接わたされます。
```

Comment:
```suggestion
    **FastAPI** は開発者の利便性のために `fastapi.testclient` と同じ `starlette.testclient` を提供します。しかし、実際にはStarletteから直接渡されています。
```
Found a typo, and the original sentence is completely same as `static-files/`'s. Better to follow the convention: https://github.com/tiangolo/fastapi/pull/2260/files#diff-cdba370c4e5a05193b80fa00e4d5c11ec671b3ba8b5f8cba145e9c9c036334c4R31